dev/archery/archery/lang/java.py

```diff
@@ -73,7 +73,7 @@ def environment(self):
             env["JAVA_HOME"] = self.java_home
 
         if self.java_options:
-            env["JAVA_OPTIONS"] = self.java_options
+            env["JDK_JAVA_OPTIONS"] = self.java_options
```

Comment:
The issue is `_JAVA_OPTIONS` being used as the environment variable instead here `JAVA_OPTIONS`? 
I am just thinking how did this work previously? Didn't picked up at all?

Reply:
It was an undocumented Java 8 option. In Java 9+, it switched to `JDK_JAVA_OPTIONS` 😅